### ISSUE TITLE
Access is denied when EncryptData is True

### ISSUE THREAD

**Issue by vasichkin040** [AgNO3/jcifs-ng#356]
I catch `jcifs.smb.SmbAuthException: Access is denied` when server share's `EncryptData` is `True`.
Is EncryptData supported or not?

**Comment by mbechler** [comment]
No, unfortunately still have not gotten around to implement SMB encryption.
